### DISCUSSION TITLE
MFXComboBox visible rows

### DISCUSSION THREAD

**Discussion by julianprimetshofer** [palexdev/MaterialFX#242] [Archived]
Hey, 
I need to work with a MFXComboBox but appearently it always only shows 3 rows. Normally it would be possible to use "comboBox.setVisibleRowCount(int)" to change the amount but it looks like MFXComboBox does not extend the JavaFx ComboBox. Is there any way to still change the amount?

**Comment by palexdev** [comment]
@Keiinmal Please see here #243
